apps/release/release/uploader.py

```
@@ -0,0 +1,501 @@
+        "Configure your local environment to authenticate with the repository "
+        "(for example, set up an SSH key or configure a GitHub token in your git "
+        "credential helper), then rerun the publish step or push the tag manually "
+        "with `git push origin {tag}`. "
+        "See docs/development/package-release-process.md#git-authentication-for-tag-pushes."
+    ).format(tag=tag_name)
+    if details:
+        message = f"{message} Git reported: {details}"
+    raise ReleaseError(message) from exc
+
+
+def _push_tag(tag_name: str, package: Package) -> None:
```

Ruff (0.14.14)

[warning] Unused function argument: `package`

(ARG001)